x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/action/DeleteJobAction.java

```diff
@@ -38,6 +38,11 @@ public static class Request extends AcknowledgedRequest<Request> {
          */
         private boolean shouldStoreResult;
 
+        /**
+         * Should user added annotations be removed when the job is deleted?
+         */
+        private boolean deleteUserAnnotations;
```

Comment:
This needs to be added to other methods in this class, e.g. `equals`, `hashCode`, `writeTo`.